task/04-arrays-tasks.js

```diff
@@ -234,7 +234,12 @@ function toArrayOfSquares(arr) {
  *   [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ] => [ 1, 3, 6, 10, 15, 21, 28, 36, 45, 55 ]
  */
 function getMovingSum(arr) {
-   throw new Error('Not implemented');
+   let result = [];
+   arr.reduce((accumulator, currentValue) => {
```

Comment:
aparantly there is a test that fobids to use loops in this task. Try to rewrite reduce so that you use the resutl of the reduce function and without **result** variable